src/components/Movies.vue

```diff
@@ -5,94 +5,51 @@
-    actors : [
-      {
-        name: 'Marlon Brando',
-        image: alex
-      },
-      {
-        name: 'Brad pit',
-        image: bradpit
-      },]
-  },
-  {
-    title: 'The Covenant',
-    image: breakingbad,
-    info: '123min | War',
-    descreption : 'The covenant is a 2006 American supernatural horror film written by J. S. Cardone, directed by Renny Harlin, and starring Steven Strait, Taylor Kitsch, Toby Hemingway, Chace Crawford, Sebastian Stan, Laura Ramsey, and Jessica Lucas. The film, despite receiving very negative reviews, was a moderate box office success.',
-    actors : [
-      {
-        name: 'Marlon Brando',
-        image: alex
-      },
-      {
-        name: 'Brad pit',
-        image: bradpit
-      },]
-  },
-  {
-    title: 'Goodfellas',
-    image: goodfellas,
-    info: '145min | Action',
-    descreption: 'Goodfellas (stylized GoodFellas) is a 1990 American crime film directed by Martin Scorsese, produced by Irwin Winkler and distributed by Warner Bros. It is an adaptation of the 1985 non-fiction book Wiseguy by Nicholas Pileggi, who co-wrote the screenplay with Scorsese. The film stars Robert De Niro, Ray Liotta, Joe Pesci, Lorraine Bracco and Paul Sorvino. It narrates the rise and fall of mob associate Henry Hill and his friends and family from 1955 to 1980.',
-    actors : [
-      {
-        name: 'Marlon Brando',
-        image: alex
-      },
-      {
-        name: 'Brad pit',
-        image: bradpit
-      },]
-  },
-  {
-    title: 'Puss in boots',
-    image: pussinBoots,
-    info: '144min | Animation',
-    descreption : "  Puss in boots is a proud and honorable fighter. He can come across as somewhat silly and has a lot of traits associated with cats but is also a smooth-talking, suave, kind-hearted, charming, and heroic character. He is also a  cat (similar to a ladies' man). He likes to tell stories and sing and dance.",
-    actors : [
-      {
-        name: 'Marlon Brando',
-        image: alex
-      },
-      {
-        name: 'Brad pit',
-        image: bradpit
-      },]
+
+  const options = {
+  method: 'GET',
+  headers: {
+    accept: 'application/json',
+    Authorization: 'Bearer eyJhbGciOiJIUzI1NiJ9.eyJhdWQiOiI1YTU2YzhmYzk2N2EzMTA1NDk2OTA3MmFhY2E2MDVmNiIsInN1YiI6IjY0YTUxYjMyMWJmMjY2MDEwNTE4MDAyMiIsInNjb3BlcyI6WyJhcGlfcmVhZCJdLCJ2ZXJzaW9uIjoxfQ.HxmHxMHAYsrdt_Q3S613yYxLtOJCHd2gYTqAfOAIY2I'
+  }
+};
+
+const getUpcomingMovie = async () =>{
+  const respponse = await fetch('https://api.themoviedb.org/3/movie/upcoming?language=en-US&page=1', options)
+  const data = await respponse.json();
+  const upcomingMovie = data.results.slice(0, 4);
```

Comment:
add guard (what if data length is lower than 4?)

Reply:
I remove the slice and now show all the data